assets/js/google-maps.min.js

```diff
@@ -0,0 +1 @@
+function initialize_map(){wpapi_gmaps.forEach(function(e,o){var n=document.getElementById("wpapi-gmap-"+o),a=new google.maps.LatLng(e.lat,e.lng),p=JSON.parse(e.style),i=!e.scrollwheel,t={center:a,zoom:Number(e.zoom),scrollwheel:i,mapTypeId:google.maps.MapTypeId.ROADMAP,styles:p},l=new google.maps.Marker({position:a}),m=e.info,g=new google.maps.InfoWindow({content:m}),s=new google.maps.Map(n,t);l.setMap(s),g.open(s,l)})}jQuery(document).ready(function(){"undefined"!=typeof wpapi_gmaps&&google.maps.event.addDomListener(window,"load",initialize_map)});
```

Comment:
__[jshint]__
* Strings must use singlequote.
* Missing semicolon.
* Expected '!==' and instead saw '!='.
* 'wpapi_gmaps' is not defined.
* 'google' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->